src/main/resources/logback-spring.xml

```diff
@@ -6,24 +6,12 @@
 
   <springProperty scope="context" name="app" source="spring.application.name"/>
 
-
-  <springProfile name="dev || stdout">
-    <appender name="consoleAppender" class="ch.qos.logback.core.helpers.NOPAppender" />
-    <appender name="logAppender" class="ch.qos.logback.core.ConsoleAppender">
-      <layout class="ch.qos.logback.classic.PatternLayout">
-        <Pattern>${LOG_PATTERN}</Pattern>
-      </layout>
-    </appender>
-  </springProfile>
-
-  <springProfile name="!(dev || stdout)">
-    <appender name="consoleAppender" class="ch.qos.logback.core.ConsoleAppender">
-      <layout class="ch.qos.logback.classic.PatternLayout">
-        <Pattern>${LOG_PATTERN}</Pattern>
-      </layout>
-    </appender>
-    <appender name="logAppender" class="com.microsoft.applicationinsights.logback.ApplicationInsightsAppender"/>
-  </springProfile>
+  <appender name="consoleAppender" class="ch.qos.logback.core.helpers.NOPAppender" />
```

Comment:
wasn't getting any output for the tests and furthermore don't think the smoketest actually runs in kubernetes with application insights enabled.